### DISCUSSION TITLE
Support for astro files

### DISCUSSION THREAD

**Discussion by docjo** [sbarex/SourceCodeSyntaxHighlight#265] [Ideas]
Hi there,
Could you explore the possibility of adding support for Astro files? 
The syntax is quite similar to HTML and JSX. Here's the documentation link for more details: 
https://docs.astro.build/en/core-concepts/astro-syntax/

Thanks a lot for considering it!

![Screenshot 2024-01-03 at 11 50 57](https://github.com/sbarex/SourceCodeSyntaxHighlight/assets/2640150/540684ae-d4f3-4324-8e85-d165ebb3ca73)

**Comment by sbarex** [comment]
Implemented on release 2.1.21.
